packages/vats/src/lib-board.js

```diff
@@ -326,7 +327,6 @@ export const prepareBoardKit = baggage => {
       /**
        * @param {string} id
        * @throws if id is not in the mapping
-       * @returns {Marshaller}
```

Comment:
Why is this being removed?

Reply:
Because `Marshaller` doesn't have the 2 new methods.

Reply:
```suggestion
       * @returns {Marshaller & {
           serializeAndStringify: (value: unknown) => string,
           parseAndDecode: (value: string) => any,
         }}
```
or better yet,
```js
// TODO Move into endo.
/**
 * @typedef {Marshaller & {
 *   serializeAndStringify: (value: unknown) => string,
 *   parseAndDecode: (value: string) => any,
 * }} ExtendedMarshaller
 */
```
```suggestion
       * @returns {ExtendedMarshaller}
```

Reply:
We typically leave return types inferred, no?

But yes, this type is needed in lots of other places, so I have started on an `ExtendedMarshaller` type. Might as well use it here, I suppose.

Reply:
> We typically leave return types inferred, no?

I don't know about typically, but my own practice is that when typing a function at all, I usually state the return type I expect explicitly, in order to catch surprises earlier.